packages/sdk/src/modules/FusePools.ts

```diff
@@ -132,6 +132,7 @@ export function withFusePools<TBase extends FuseBaseConstructor>(Base: TBase) {
         comptroller,
         name,
         totalLiquidityNative,
+        totaAvailablelLiquidityNative,
```

Comment:
```suggestion
        totalAvailablelLiquidityNative,
```

Reply:
typo

Reply:
and there's still a typo (doule L) haha. Will fix